crates/ra_hir_ty/Cargo.toml

```diff
@@ -27,8 +27,8 @@ test_utils = { path = "../test_utils" }
 
 scoped-tls = "1"
 
-chalk-solve =   { git = "https://github.com/rust-lang/chalk.git", rev = "329b7f3fdd2431ed6f6778cde53f22374c7d094c" }
-chalk-ir =      { git = "https://github.com/rust-lang/chalk.git", rev = "329b7f3fdd2431ed6f6778cde53f22374c7d094c" }
+chalk-solve = "0.11"
+chalk-ir = "0.11"
```

Comment:
is chalk developed slowly enough today to make crates.io publishing to be worth it?

Reply:
@pksunkara built an automatic release setup for Chalk, so we should still be getting updates (almost) as soon as we want them.